Sources/Build/BuildPlan.swift

```diff
@@ -663,17 +662,13 @@ public final class SwiftTargetBuildDescription {
         guard let bundlePath = self.bundlePath else { return }
 
         let stream = BufferedOutputByteStream()
-
-        let mainPath: AbsolutePath =
-            AbsolutePath(Bundle.main.bundlePath).appending(component: bundlePath.basename)
-
         stream <<< """
         import class Foundation.Bundle
 
         extension Foundation.Bundle {
             static var module: Bundle = {
-                let mainPath = "\(mainPath.asSwiftStringLiteralConstant)"
-                let buildPath = "\(bundlePath.asSwiftStringLiteralConstant)"
+                let mainPath = Bundle.main.bundlePath + "\(bundlePath.basename.asSwiftStringLiteralConstant)"
```

Comment:
Is the `bundlePath` guaranteed to end with a path separator on all platforms, or might one need to be conditionally inserted here?

Reply:
@abertelrud Good point, thanks! The old code also appended a slash. I've added this back!

Reply:
@abertelrud Actually, I'm now using `Bundle.main.bundleURL.appendingPathComponent`. This moves the responsibility for path separators to `Foundation.URL`. Using a forward-slash might break Windows, since the path separator is a backslash there.
Hope this is ok?

Reply:
Thanks @ffried, I think that's even better, especially for Windows compatibility.